sanic/utils.py

```diff
@@ -0,0 +1,69 @@
+from os import environ as os_environ
+from re import findall as re_findall
+from importlib.util import spec_from_file_location, \
+                           module_from_spec
+from typing import Union
+
+
+
+def str_to_bool(val):
+    """Takes string and tries to turn it into bool as human would do.  
+
+    If val is in case insensitive ("y", "yes", "yep", "yup", "t", "true", "on", "enable", "enabled", "1") returns True.  
+    If val is in case insensitive ("n", "no", "f", "false", "off", "disable", "disabled", "0") returns False.  
+    Else Raise ValueError."""
+
+    val = val.lower()
+    if val in ("y", "yes", "yep", "yup", "t", "true", "on", "enable", "enabled", "1"):
+        return True
+    elif val in ("n", "no", "f", "false", "off", "disable", "disabled", "0"):
+        return False
+    else:
+        raise ValueError("Invalid truth value %r" % (val,))
```

Comment:
Can we use here `f-strings`?

Reply:
Thank You for this hint.  

In general maybe we should use f-string across Sanic ???

Reply:
@myuz introduced this PR a few cycles back: https://github.com/huge-success/sanic/pull/1793